cmd/interval/add/add.go

```diff
@@ -54,6 +54,23 @@ const baseIntervalTemp = `{
 }
 `
 
+const IntervalStartUsage = "Interval Start time in format YYYYMMDD'T'HHmmss."
+const IntervalEndUsage = "Interval End time in format YYYYMMDD'T'HHmmss."
+const CronIntervalUsage = "Styled regular expression indicating " +
+	"how often the action under interval should occur. Use either runOnce, frequency or cron and not all."
+const RunOnceIntervalUsage = "RunOnce - boolean indicating that this interval " +
+	"runs one time - at the time indicated by the start."
+const FrequencyIntervalUsage = "Interval frequency - indicated how frequent the " +
```

Comment:
s/indicated how frequent the/indicates how frequently the

Reply:
fixed